### PR TITLE
Fix Create Question Paper CSS and JS

### LINKED ISSUE #530: Create Question Paper not working due to CSS and JS update in the new UI
Cannot create question paper!
The new UI merged has CSS and JS libraties updated.
The design question paper creation CSS and JS are not modified as per the update. Hence not working.
The Student interface seems to be working okay but the moderator interface is breaking at few places.
So the moderator interface needs to be checked thoroughly.

### PR DESCRIPTION
**Pull request by CruiseDevice**
- Fixes nav-tabs navigation.
- Proper alignment of dropdowns.

### CODE REVIEW

**Comment by ankitjavalkar** [comment]
Looks good. This can be merged.

File: yaksh/templates/yaksh/design_questionpaper.html [hunk 1]
```diff
@@ -34,33 +34,36 @@
...
-    <li id="finish-tab">
-    <a href="#finish" id="finished">
-        STEP 3<br>
-        Finish
-    </a></li>
-</ul>
-
+    <div class = "tab-base">
+        <ul class="nav nav-pills tabs">
+        <li class="" id="fixed-tab">
```
Comment (maheshgudi) [inline yaksh/templates/yaksh/design_questionpaper.html]: Empty list class. Can be removed

File: yaksh/static/yaksh/js/bootstrap-tabs.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
+/* ========================================================
+ * bootstrap-tabs.js v1.4.0
+ * http://twitter.github.com/bootstrap/javascript.html#tabs
```
Comment (prathamesh920) [inline yaksh/static/yaksh/js/bootstrap-tabs.js]: We do not need this additional JS!
Tabs should work with the updated bootstrap JS and CS.
You need to update the HTML template along with custom JS and CSS (for design question paper) to work with the updated bootstrap.

**Comment by CruiseDevice** [comment]
> Hello @CruiseDevice! Thanks for updating the PR.
> 
> * In the file [`yaksh/views.py`](https://github.com/FOSSEE/online_test/blob/74b69f565a3142902695a08b815c2b9a214b6662/yaksh/views.py), following are the PEP8 issues :
> 
> > [Line 908:80](https://github.com/FOSSEE/online_test/blob/74b69f565a3142902695a08b815c2b9a214b6662/yaksh/views.py#L908): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (80 > 79 characters)
> > [Line 909:13](https://github.com/FOSSEE/online_test/blob/74b69f565a3142902695a08b815c2b9a214b6662/yaksh/views.py#L909): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> > [Line 1292:17](https://github.com/FOSSEE/online_test/blob/74b69f565a3142902695a08b815c2b9a214b6662/yaksh/views.py#L1292): [W605](https://duckduckgo.com/?q=pep8%20W605) invalid escape sequence '*'

@prathamesh920 this pep8 issue is fixed in this PR. Should i keep it like this or make changes and push again?

**Comment by ankitjavalkar** [comment]
@prathamesh920 @CruiseDevice We have made sure that bootstrap-tabs.js is deprecated and not used anymore. We have maintained everything as is so that nothing else should fail unexpectedly

File: yaksh/templates/yaksh/design_questionpaper.html [hunk 3]
```diff
@@ -16,8 +16,8 @@
 
 {% block script %}
     <script src="{{ URL_ROOT }}/static/yaksh/js/jquery-1.9.1.min.js"></script>
```
Comment (prathamesh920) [inline yaksh/templates/yaksh/design_questionpaper.html]: jquery-1.19.1 is deleted!
Also test is failing due to this.
Kindly fix this.

**Comment by ankitjavalkar** [comment]
@prathamesh920 The Jquery v1.9.1 file has been replaced. The JQuery version in the create_questionpaper file also stays the same.

This is because I asked @CruiseDevice to create a seperate PR/Changeset for changing all the references from JQuery 1.9.1 to JQuesry 3.3.1 since this may lead to unexpected changes/breakages in place we may not expect.

Rest assured, we will test it (along with the JQuery UI change) and merge it, but separately.

**Comment by prathamesh920** [comment]
LGTM

**Pull request merged**